esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3357,14 +3354,36 @@ bool CFileSprayEx::getDropZoneFiles(IEspContext &context, const char* dropZone,
     return true;
 }
 
-//This method returns all dropzones and, if NetAddress and Path specified, returns filtered list of files.
+void CFileSprayEx::getServersInDropZone(const char *dropZoneName, IArrayOf<IConstTpDropZone> &dropZoneList, bool ECLWatchVisibleOnly, StringArray &serverList)
```

Comment:
trivial: unusual to have a variable that starts with a capital letter.

Reply:
putting 'is' on the front may be a simple fix...